db/migrate/20200617193919_create_product_option_types.rb

```diff
@@ -0,0 +1,8 @@
+class CreateProductOptionTypes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :product_option_types do |t|
+
```

Comment:
mira como funciona el many to many esta tabla debe de tener referencias https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association mira por ejemplo la migracion de appointments